projects/netgrif-components-core/src/lib/task-content/task-content/abstract-task-content.component.spec.ts

```diff
@@ -509,7 +509,7 @@ describe('AbstractTaskContentComponent', () => {
             expect(component.dataSource[0].content.length).toEqual(1);
             expect(component.dataSource[0].content[0].type).toEqual(FieldTypeResource.BOOLEAN);
 
-            expect(component.dataSource[1].content.length).toEqual(1);
+            expect(component.dataSource[1].content.length).toEqual(9);
```

Comment:
these tests should be investigated more closely. The behavior of non-dashboard task refs should remain the same and there should be no reason to change any of the expectations in this FILE